packages/client/package.json

```diff
@@ -83,7 +83,14 @@
       "import": "./wasm.js",
       "default": "./wasm.js"
     },
-    "./*": "./*"
+    "./*.js": "./*.js",
+    "./*.d.ts": "./*.d.ts",
+    "./*": {
+      "types": "./*.d.ts",
+      "require": "./*.js",
+      "import": "./*.js",
+      "default": "./*.js"
+    }
```

Comment:
This is to fix `https://github.com/prisma/prisma/issues/22875`. The exports map made imports such as `@prisma/client/runtime/library` impossible and needed a `.js` extension to work. Now it allows both with and without `.js`.

An exports map reads from top to bottom, with most specific keys first taking precedence over less specific ones below. So here we are basically saying: "allow imports in `.js`, `.d.ts`, and in any other case where it is not specified (eg. `/runtime/library`), then add the `.js` extension automatically.

It will allow people to continue using internals for the time being, we can decide if we want to prevent this in Prisma 6.

Reply:
Updates https://github.com/newrelic/node-newrelic/issues/1984#issuecomment-1918097671